### ISSUE TITLE
where can you change your username and values

### ISSUE THREAD

**Issue by olpexxx** [kangoka/tiktok-autoviews#3]

**Comment by kangoka** [comment]
First of all, you need the script installed, then goto Dashboard Tampermonkey by clicking Tampermonkey icon on extension. After that, just click the name of the script and there you can edit change your username and value.

**Comment by olpexxx** [comment]
it still puts 1k views on the video

**Comment by olpexxx** [comment]
yes i know that but i change the 1000 views to whatever and still put the 5 value in i still get 1k

**Comment by kangoka** [comment]
Ah right, thank you for your report. I will fix it ASAP.

**Comment by olpexxx** [comment]
thank you, cuz this script is fire👍 🔥

**Comment by kangoka** [comment]
Hi, after I checked, you need to change this line
https://github.com/kangoka/tiktok-autoviews/blob/60954e78bc915ecbc0d49763ca37d609406f37c8/script.js#L60
